apps/meteor/client/views/marketplace/AppsProvider.tsx

```diff
@@ -303,6 +352,10 @@ const AppsProvider: FC = ({ children }) => {
 				return;
 			}
 
+			if (privateApp) {
+				dispatchPrivateApps({ type: 'update', app: privateApp, reload: fetch });
```

Comment:
No `return`? It's supposed to dispatch twice?

Reply:
two different lists